src/structures/Player.ts

```diff
@@ -198,7 +197,7 @@ export class Player {
                         clearInterval(connectInterval);
                         return reject(new ChiroError(`Player has been disconnected but has been assigned to play!`));
                     }
-                }, 50000);
+                }, 10000);
```

Comment:
It's better to make the interval 1000. Ideally the ready event is received in 1-2 sec. Putting it to 10sec will make user wait for 10sec to play song

Reply:
That code is outdated btw.

Reply:
why is this interval here? What's the purpose of doing this??

Reply:
Thats an outdated code.